Source/Modules/fortran.cxx

```diff
@@ -3562,9 +3562,9 @@ int FORTRAN::classDirectorMethod(Node *n, Node *classn, String *super) {
 
     // Create local variables that become arguments to fortran
     {
-      String *cdecl = SwigType_lstr(ctype, imarg);
-      Wrapper_add_localv(cppfunc, imarg, cdecl, NULL);
-      Delete(cdecl);
+      String *cdecl2 = SwigType_lstr(ctype, imarg);
+      Wrapper_add_localv(cppfunc, imarg, cdecl2, NULL);
+      Delete(cdecl2);
```

Comment:
I think this change is accidental?

Reply:
Ah, that was not an accident, but I definitely should have broken that out into a separate pull request.  When compiling Swig from source with VS2022 (though I suspect other versions of MSVC will encounter the same issue), cdecl is a keyword and cannot be used as a variable name.  I just added a 2 to avoid the conflict, but if you see a logical name that avoids the conflict, it could be improved.

Reply:
Aha, I actually encountered this error much later when I was updating the upstream SWIG merge request. It's fixed on the `fortran` branch and I'm cherry-picking onto master now.